docs/usage.rst

```diff
@@ -452,6 +452,30 @@ functions that leverage this capability; builtins including, but not limited to:
 (for examples, see :ref:`sec_max_min`) and :func:`sorted`.
 
 
+Version Equality
```

Comment:
Could we have the title parallel to the others? I think, there is a verb missing. So maybe something like this:

```suggestion
Determining Version Equality
```